### DISCUSSION TITLE
Impossible to connect with New Galaxy S22+

### DISCUSSION THREAD

**Discussion by BreizhBigoud** [AppGround-io/bluetooth-keyboard-and-mouse-support#267] [Bugs & Problems]
Hello,
I was using the app on my Xiaomi Mi Mix 2S with m'y Raspberry Pi4 and it was working really well.
I've juste bought a new device, Galaxy S22+.
Tonight i was trying to connect m'y device to keep on using the app, and i never succeeded to connect it to m'y Raspberry.
Did you hear about this issue ?
Sorry for my pour english.... 😭
Thx for tour Hello
BreizhBigoud

**Comment by rockymehta1** [comment]
I am using an S22 ultra to connect to a Samsung tablet and Mac Book with no issues

**Comment by e100** [comment]
I have a Samsung Note 10+ and had issues pairing with my PI 4.
After pairing, removing from the phone and PI and repairing many times I finally got it to work.

Not sure if it was the solution but during the pair that worked I accepted the pair on the PI first then quickly accepted on the phone.
Works great once paired!
